test_cases/september_test_case.json

```diff
@@ -1102,7 +1106,16 @@
                     }
                 ],
                 "Label": "POTUS"
-            }
+            },
+            "ExternalIdentifier": [
```

Comment:
This implementation unfortunately deviates from the one in the August EDF.

* In the August EDF, [the `ExternalIdentifier` is on the `Contest`](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/bd1fa7881a52ae530fbc86a004e0221d1c9ed0a6/test_cases/august_test_case.json#L81-L119)
* In this PR, it's being added to the `Office`

Markit is configured to look on the `Contest`, and so this update results in a failure to parse - thanks to my updated parsing logic that expects it there.

I went looking in the [profile](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/main/profile/profile.org), but couldn't find any documentation about how we agreed to do this.

Conceptually, I think it makes more sense for it to be applied to the contest. What do you think?

Reply:
You're right. I'll fix that.

Reply:
@mattbeattie , the CandidateContests now have ExternalIdentifiers; please confirm

Reply:
Resolving this thread as this has been completed.